File: src/wsb/po/dwa/PrimeNumbers.java
package wsb.po.dwa;

import java.util.Scanner;

public class PrimeNumbers {
    public static boolean isPrime(int n) {
        if (n < 2) return false;
        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) return false;
        }
        return true;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Podaj liczbę: ");
        int n = scanner.nextInt();
        scanner.close();

        System.out.print("Liczby pierwsze: ");
        for (int i = 2; i < n; i++) {
            if (isPrime(i)) {
                System.out.print(i + " ");
            }
        }
    }
}

File: _Python/dwa/prime_numbers.py
def is_prime(n):
    if n < 2:
        return False
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False
    return True

n = int(input("Podaj liczbę: "))
primes = [i for i in range(2, n) if is_prime(i)]
print("Liczby pierwsze:", primes)


for i in range(2,n):
    if is_prime(i):
        print(str(i) + " ",end=", ")
            